core-field-editors/src/textarea/index.js

```diff
@@ -0,0 +1,40 @@
+import throttle from 'lodash/throttle'
+import createUpdater from '../_shared/input-updater'
+import mkBoot from '../_shared/boot'
+
+export const boot = mkBoot(create)
+
+export default function create (api, opts = {}) {
+  const document = opts.document || window.document
+
+  const textarea = document.createElement('textarea')
+  textarea.classList.add('cf-form-input')
+  const field = api.field
+  const updateInput = createUpdater(textarea, document)
+
+  field.onValueChanged(function (val) {
+    // Might be `null` or `undefined` when value is not present
+    updateInput(val || '')
+  })
+
+  field.onIsDisabledChanged(function (isDisabled) {
+    textarea.disabled = isDisabled
+  })
+
+  field.onSchemaErrorsChanged(function (errors) {
+    if (errors && errors.length > 0) {
+      textarea.setAttribute('aria-invalid', true)
+    } else {
+      textarea.removeAttribute('aria-invalid')
+    }
+  })
+
+  const updateValue = throttle(function () {
```

Comment:
Can we extract throttling setup to a helper function that could be used across editors (for consistent behavior).

We could also consider attaching events in such function (so it'd be `fn(el, onupdate)`). I think the code below applies to all textual editors.